viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -67,9 +67,9 @@ ::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
 
     // Set pose for our response
     Pose *myPose = response->mutable_pose();
-    myPose->set_x(pos_vector.x());
-    myPose->set_y(pos_vector.y());
-    myPose->set_z(pos_vector.z());
+    myPose->set_x(pos_vector.x() * 1000);
```

Comment:
[note to self] Cartographer is returning the values in meters (I couldn't see exactly where, but there are plenty of results in the cartographer repo for `git grep "[^para]meter"` and none for "millimeter" or "mm" besides things like symmetric, jammy, dummy, etc.

The Pose proto is defined here: https://github.com/viamrobotics/api/blob/a51d6714c844627e7caaea67c6471a015ccfec85/proto/viam/service/slam/v1/slam.proto#L53

And used by slam here: https://github.com/viamrobotics/api/blob/a51d6714c844627e7caaea67c6471a015ccfec85/proto/viam/service/slam/v1/slam.proto#L53